### ISSUE TITLE
The RNN learns nothing on trainset_B

### ISSUE THREAD

**Issue by nalanyu2000** [jambo6/sepsis_competition_physionet_2019#2]
I established the training dataset_A and dataset_B respectively by the code provided instead of using original dataset_AB. I evaluated the RNN example on dataset_A and dataset_B. I found that RNN learned nothing  and predict all data as 0 on dataset B. I tried some positive weight on loss function but obtained unsatisfactory results. I wonder how to make it work.
